interfaces/builtin/bluez.go

```diff
@@ -225,3 +225,13 @@ func (iface *BluezInterface) AutoConnect(*interfaces.Plug, *interfaces.Slot) boo
 	// allow what declarations allowed
 	return true
 }
+
+func (iface *BluezInterface) ValidatePlug(plug *interfaces.Plug, attrs map[string]interface{}) error {
```

Comment:
@zyga Ok, I made ValidatePlug/Slot optional in the PR #3120 as this is where it's used (also improved a test there to make sure it's called when it should). ValidatePlug/Slot is removed from base Interface in core.go completely and I've removed it from all interfaces which have no attributes (so there is no potential use for Validate* to exists atm); I've left empty stubs for Validate* in interfaces which can potentially make use of attributes from hooks.